app/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
 
     defaultConfig {
         applicationId "com.kamwithk.ankiconnectandroid"
-        minSdk 21
+        minSdk 26
```

Comment:
Does this mean the code is now able to use java 8 features, such as lambda expressions and streams?

Reply:
I'm pretty sure Java 8 doesn't rely on minSdk. If you see SettingsActivity.java, there are a few lambda expressions in there already.